tutorial/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ visp_add_subdirectory(imgproc/contour                      REQUIRED_DEPS visp_co
 visp_add_subdirectory(imgproc/contrast-sharpening          REQUIRED_DEPS visp_core visp_io visp_gui visp_imgproc)
 visp_add_subdirectory(imgproc/count-coins                  REQUIRED_DEPS visp_core visp_io visp_gui visp_imgproc)
 visp_add_subdirectory(imgproc/flood-fill                   REQUIRED_DEPS visp_core visp_io visp_gui visp_imgproc)
+visp_add_subdirectory(imgproc/hough-transform              REQUIRED_DEPS visp_core visp_gui visp_imgproc)
```

Comment:
Since `vpImageMedian.*` files are removed, you can remove `visp_imgproc` dependency.

Reply:
Done in  9c20c0c21

Reply:
Put it back in dc114ddf8 because vpCHT is defined in visp_imgproc